scripts/test-sim-dev.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+CONTAINER_NAME=storj_sim_postgres
+docker run -d --rm -p 5433:5432 --name $CONTAINER_NAME postgres:9.6
```

Comment:
It would be really nice to be able to collect logs from this postgres somewhere, either live or before the container is destroyed. maybe the best way would be to do `docker logs $CONTAINER_NAME > $tmpfile` in the cleanup function? or perhaps starting up something like `tail -f $(docker inspect --format '{{.LogPath}}' $CONTAINER_NAME) > $tmpfile &` here?

And then we could add `-c log_min_duration_statement=0` to this invocation, and get nice logs about every query that goes through, for those times when we care.

Reply:
ok, I will try to combine it somehow :)

Reply:
live logging done